backends/advanced/src/advanced_omi_backend/routers/modules/audio_routes.py

```
@@ -12,10 +12,24 @@
 from advanced_omi_backend.controllers import audio_controller
 from advanced_omi_backend.models.user import User
 from advanced_omi_backend.app_config import get_audio_chunk_dir
+from advanced_omi_backend.utils.gdrive_audio_utils import download_audio_files_from_drive
 
 router = APIRouter(prefix="/audio", tags=["audio"])
 
 
+@router.post("/upload_audio_from_url")
+async def upload_audio_from_drive_folder(
+    drive_folder_id: str = Query(...,alias="url", description="Google Drive Folder ID containing audio files (e.g., the string after /folders/ in the URL)"),
+    current_user: User = Depends(current_superuser),
```

Ruff (0.14.7)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)